src/ChannelServer/World/Dungeons/DungeonManager.cs

```diff
@@ -121,7 +121,7 @@ private Dungeon CreateDungeon(string dungeonName, int itemId, Creature creature)
 			{
 				// Create new dungeon if there is non yet or it's Tuesday
 				var existing = this.Get(a => a.Name == dungeonName && a.ItemId == itemId);
-				if (existing == null || ErinnTime.Now.Month == 2 || ChannelServer.Instance.Conf.World.PrivateDungeons)
```

Comment:
DeleteIT:
>Tuesday does not create new dungeons with the same item, with the previous dungeon made with that same item open.
Try it next Tuesday, if you don't believe me, I've already verified this.

I need to (and will) move/change that check.

In all seriousness though, this is a feature that could be added to the conf, or, a check could be made so that you're returned to any dungeon that you were a creating member of (in the case of DCing/crashing) otherwise, Tuesday is a day of "Okay, we lost [name]. Should we start again..?"

It also makes Tuesday less accessible to mass raids, like, when people used to have 1G raids on Peaca.